### PR TITLE
Fix the dev server

### PR DESCRIPTION
**Pull request by gartz**
https://github.com/gartz/pushStateTree/pull/21 introduced a bug in the dev-server.

### CODE REVIEW

File: demo/devserver.js [hunk 1]
```diff
@@ -0,0 +1 @@
+window.PushStateTree = require('push-state-tree');
```
Comment (romuloab) [inline demo/devserver.js]: Are you sure about that? Wouldn't it be better to require when needed?

File: karma.conf.js [hunk 2]
```diff
@@ -26,20 +30,38 @@ yargs.options({
...
 if (WATCH) {
   let port = argv['dev-port'] || 8080;
 
   const webpackDevConfig = Object.assign({
-    cache: true,
-    devtool: 'hidden-source-map'
-  }, webpackConfig);
+    resolve: {}
+  }, webpackConfig, {
+    devtool: FAST ? 'cheap-module-eval-source-map' : 'inline-source-map',
```
Comment (romuloab) [inline karma.conf.js]: why? can't you just stick to one?

Comment (romuloab) [inline karma.conf.js]: also, shouldn't this be documented somewhere?

File: demo/devserver.js (hunk 1 above)
Comment (gartz) [inline demo/devserver.js]: I think it's a good point, need to add some comment here to explain why I'm doing this.

**Comment by romuloab** [comment]
done reviewing

I'm no expert in webpack/karma, so I'm afraid I can't review better :\

**Pull request merged**